packages/uswds-core/src/styles/tokens/color/shortcodes-color-state.scss

```diff
@@ -30,6 +30,11 @@ $tokens-color-state: (
     set-theme-color(settings.$theme-color-disabled-light, no-warn),
   "disabled": set-theme-color(settings.$theme-color-disabled, no-warn),
   "disabled-dark": set-theme-color(settings.$theme-color-disabled-dark, no-warn),
+  "disabled-text": set-theme-color(settings.$theme-color-disabled-text, no-warn),
+  "disabled-text-reverse":
+    set-theme-color(settings.$theme-color-disabled-text-reverse, no-warn),
+  "text-on-disabled":
```

Comment:
> **Warning**
> I could not add "text-on-disabled" here because it caused an error for not following the standard `color`-`grade` naming format. There are a couple of paths forward here if we want to make this consistent. 
> 
> One option would be to give `$theme-color-text-on-disabled` (and maybe the other new settings) a name that fits better with the current state token format. (For this example, we could potentially appropriate `disabled-dark` because it is no longer used.) 
>
> Alternatively, I wonder if these new disabled settings might be better described as component tokens since they deviate in both purpose and format from other state colors. 
>
> Curious to hear what you think.